woocommerce/wc-admin
# Update dependencies in prep for Composer 2.x.

This PR seeks to ready our repository for the upcoming Composer 2.x update.

See: pc2DNy-3j-p2 for more information.

Some packages needed updating to facilitate compatibility with Composer 2, but I experienced errors with the latest Jetpack Autoloader which is why it gets pinned to `2.3.0` in this PR.

### Detailed test instructions:

- Verify installation works on Composer 1.x:
- `rm -rf vendor && composer install`
- Check that app works (no PHP Fatals missing classes, etc)
- Verify installation works on Composer 2.x:
- `composer self-update --preview`
- `rm -rf vendor && composer install`
- Check that app works (no PHP Fatals missing classes, etc)




### Changelog Note:



N/A
